docs/gl_objects/users.rst

```diff
@@ -192,6 +192,31 @@ Revoke (delete) an impersonation token for a user::
     i_t.delete()
 
 
+User projects
+=========================
+
+References
+----------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.UserProject`
+  + :class:`gitlab.v4.objects.UserProjectManager`
+  + :attr:`gitlab.v4.objects.User.projects`
+
+* GitLab API: https://docs.gitlab.com/ee/api/projects.html#list-user-projects
+
+List visible projects owned by the user::
```

Comment:
Feels to me like the upstream docs are a bit contrived there, could we directly state it's only projects in the namespace and not those owned by the user?
 
```suggestion
List visible projects in the user's namespace::
```

Reply:
Sure!  Done.